src/steps/lead-merge.ts

```diff
@@ -8,10 +8,12 @@ import { isNullOrUndefined } from 'util';
 
 export class MergeLeadsStep extends BaseStep implements StepInterface {
 
-  protected stepName: string = 'Merge Marketo Leads';
+  protected stepName: string = 'Merge Marketo leads';
   // tslint:disable-next-line:max-line-length
   protected stepExpression: string = 'merge marketo lead (?<losingEmail>.+\@.+\..+) into marketo lead (?<winningEmail>.+\@.+\..+)';
   protected stepType: StepDefinition.Type = StepDefinition.Type.ACTION;
+  protected actionList: string[] = ['misc'];

```

Comment:
not sure how users will know to look under 'misc' for this step, maybe on the front end we can show misc steps in other categories like 'create' or 'update' just for ease of use.